Jenkinsfile

```diff
@@ -78,7 +78,7 @@ def stablePresubmit(gitUtils, bazel, utils) {
     sh('ln -s ~/.kube/config platform/kube/')
     stage('Integration Tests') {
       timeout(30) {
-        sh("bin/e2e.sh -count 10 -debug -tag debug${gitUtils.GIT_SHA} -v 2")
```

Comment:
it won t be the same SHA as one SHA is from a PR to master, while the second will be the SHA after merge